pycloudlib/lxd/instance.py

```diff
@@ -354,7 +354,6 @@ def _wait_for_cloudinit(self, *, raise_on_failure: bool):
                     # lxd vms. However, if we wait a moment before trying
```

Comment:
Does the retrying we're implementing elsewhere mean that this retrying is no longer necessary?

Reply:
Unfortunately no. This is related to that second problem I commented regarding `cloud-init status --wait --long` returning `-1` in the middle of the operation. We still don't know what is causing that issue